configure.ac

```diff
@@ -6,6 +6,8 @@ define(_CLIENT_VERSION_REVISION, 2)
 define(_CLIENT_VERSION_BUILD, 2)
 define(_CLIENT_VERSION_IS_RELEASE, true)
 define(_COPYRIGHT_YEAR, 2017)
+define(_COPYRIGHT_HOLDERS,[The %s developers])
+define(_COPYRIGHT_HOLDERS_SUBSTITUTION,[Dash Core])
 AC_INIT([Dash Core],[_CLIENT_VERSION_MAJOR._CLIENT_VERSION_MINOR._CLIENT_VERSION_REVISION],[https://github.com/dashpay/dash/issues],[dashcore])
```

Comment:
Needs `[https://dash.org/]` like in original PR https://github.com/bitcoin/bitcoin/pull/7192/files#diff-67e997bcfdac55191033d57a16d1408aR11

Reply:
Fixed and squashed